src/controllers/midi/portmidienumerator.cpp

```diff
@@ -1,13 +1,15 @@
 #include "controllers/midi/portmidienumerator.h"
 
 #include <portmidi.h>
+#include <qlatin1stringview.h>
```

Comment:
What is this used for? Looks like it doesn't exist on MacOS

Reply:
whoops, yeah clangd tries its best to add the relevant includes but its gets it wrong and I don't notice it. I'll remove and force push (and look into disabling that for certain headers).

Reply:
done

Reply:
fyi in case any1 else stumbles over this, see https://clangd.llvm.org/guides/include-cleaner#adjusting-behavior and use `IgnoreHeaders: q.*.h` until [QTBUG-124159](https://bugreports.qt.io/browse/QTBUG-124159) is fixed.